core/src/main/kotlin/utils/Extensions.kt

```diff
@@ -43,3 +46,85 @@ fun ConfigManager.createKeycloakClientConfiguration(): KeycloakClientConfigurati
         subjectClientId = getString("keycloak.subjectClientId")
     )
 }
+
+/**
+ * Get all [PluginConfig]s from a [CreateOrtRun] object. The result is a map from [PluginType]s to maps of plugin IDs to
+ * [PluginConfig]s. Some job configs allow to enable plugins without addintional configuration, in this case `null` is
+ * used as the value for the plugin ID in the map.
+ */
+fun CreateOrtRun.getPluginConfigs(): Map<PluginType, Map<String, PluginConfig?>> =
```

Comment:
Question: Wouldn't an empty `PluginConfig` instead of a `null` value be easier to handle for consumers?